templates/zerver/help/import-your-settings.md

```diff
@@ -0,0 +1,38 @@
+# Import your settings
+
+When you create a Zulip account using an email address already associated with
+an account in another Zulip organization, you can import your user settings
+from an existing account. It's a convenient way to preserve the user settings
+that you've already customized.
+
+!!! tip ""
+    Settings that may not apply to all organizations, such as custom profile
+    fields, will not be imported.
+
+The import will include your:
+
+- [Name and avatar]({{ root_domain_uri }}/#settings/profile)
+- [Privacy settings]({{ root_domain_uri }}/#settings/account-and-privacy)
+- [Display settings]({{ root_domain_uri }}/#settings/display-settings)
+- [Notification settings]({{ root_domain_uri }}/#settings/notifications)
+- Hotspots/tutorial state
+
+### Import your settings
+
+{start_tabs}
+
+1. _(recommended)_ [Review](/help/review-your-settings) the settings you plan to import.
```

Comment:
I think it might be better to have a "Review the settings" after importing -- do we actually think it's important to review this before creating their account? I suppose the main situation I can see is if you have 17 organizations and aren't sure which is best.

Reply:
Hm, yeah maybe it would be better after importing:

<img width="538" alt="image" src="https://user-images.githubusercontent.com/2343554/189466086-17e985d0-c3b2-46e6-8147-0b79dc556a03.png">

Reply:
Yeah, I think that's better.